src/DI/CzechPostExtension.php

```diff
@@ -17,10 +19,12 @@ class CzechPostExtension extends CompilerExtension
 	/** @var mixed[] */
 	protected $defaults = [
 		'http' => [
-			'base_uri' => null,
-			'auth' => [],
+			'base_uri' => 'https://online.postservis.cz/',
+			'auth' => ['', ''],
```

Comment:
Jestli ten auth je povinnej, tak by se mel zvalidovat v loadConfiguration, jestli neni povinnej, tak NULL.

Reply:
validuje se to pri vytvareni te servicy v `assertUsernamePassword()`. V loadConf uz bych to neduplikoval.